src/proto.rs

```diff
@@ -180,13 +180,15 @@ fn interpolate_tokens(
 
     // Avoid detecting musl unless requested
     if value.contains("{libc}") {
+        let libc = HostLibc::detect(env.os);
```

Comment:
There's a new `env.libc` you can use here, since detect won't work in wasm.

Reply:
Ahh, didn't realize the implementation of is_musl was different. Glad you caught it.